odk/extension/src/main/java/io/samagra/odk/collect/extension/handlers/ODKHandler.kt

```diff
@@ -66,113 +66,5 @@ class ODKHandler @Inject constructor(
         CoroutineScope(Job()).launch{ ConfigHandler(application).reset(listener) }
     }
 
-    override fun openForm(formId: String, context: Context) {
```

Comment:
Please remove the unused objects that are not required in this class anymore, for example, `formsDatabaseInteractor`